client/src/App.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import theme from './theme';
+import GlobalStyles from './GlobalStyles';
 
 function App() {
-  return <div>Under Construction</div>;
+  return (
+    <ThemeProvider theme={theme}>
+      <GlobalStyles></GlobalStyles>
```

Comment:
```suggestion
      <GlobalStyles />
```

Just a minor thing. This can be shortened.